### DISCUSSION TITLE
atom type

### DISCUSSION THREAD

**Discussion by hujianyuxhl** [deepmodeling/deepmd-kit#2028] [Q&A]
I have three atom type,Cu , O and Zn, which is "type_map":     ["Cu", "O","Zn"] in input.json. So , "Cu", "O" and "Zn" is 0, 1, 2 in the type.raw in training data if the three atom types exist in one system. But if another system  contains "O" and  "Zn", where the type_map.raw  is "O" and  "Zn" , is it  0  1 , or 1  2 in the type.raw?

**Comment by njzjz** [comment]
`type.raw` should always map to `type_map.raw` when both files exist.
